client/js/array-util.js

```diff
@@ -1,12 +1,24 @@
+//Returns array from fromNum to toNum that is sequential in order, both increasing and decreasing
 const getRange = function(fromNum, toNum) {
```

Comment:
Doesn't look like it worked. Did you save each file after making those changes? git will register a change to the file if you make those changes and then save; that's how you can commit the change and push it back up here.